bsp/esp_bsp_generic/src/esp_bsp_generic.c

```diff
@@ -194,8 +194,8 @@ static led_indicator_gpio_config_t bsp_leds_gpio_config[] = {
 static const led_strip_config_t bsp_leds_rgb_strip_config = {
     .strip_gpio_num = CONFIG_BSP_LED_RGB_GPIO,   // The GPIO that connected to the LED strip's data line
     .max_leds = BSP_LED_NUM,                  // The number of LEDs in the strip,
-    .led_pixel_format = LED_PIXEL_FORMAT_GRB, // Pixel format of your LED strip
     .led_model = LED_MODEL_WS2812,            // LED strip model
+    .color_component_format = LED_STRIP_COLOR_COMPONENT_FMT_GRB,  // LED color component format
```

Comment:
```suggestion
```

Reply:
leave this field empty means, i.e. init the field to `0`, the led_strip will fall-back to use the default format "GRB" as well.

Then this esp_bsp_generic package can work both with led_strip version2 and led_strip version3.